larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -731,41 +674,15 @@ void runTest(const TestParam& param) {
   // Invoke the op and test that the output is correct.
   m_lce.Invoke();
   test_lce_op_output(m_lce.GetOutput(), m_builtin.GetOutputShape(),
-                     builtin_output, builtin_output_tensor.zero_point);
+                     builtin_output, output_tensor.zero_point,
+                     output_tensor.scale);
 }
 
-// Three input types: int8, TBitpacked (int32), float
-// Three output types: int8, TBitpacked (int32), float
-// Thats 9 combinations, but float -> int8 and int8 -> float are never used so
-// that leaves 7 combinations.
+TEST_P(BConv2DOpTest, Bitpacked) { runTest<TBitpacked>(TestParam(GetParam())); }
```

Comment:
How do you feel about being more explicit with the test name here (also `FloatOutput` and `Int8Output` for the cases below)?

```suggestion
TEST_P(BConv2DOpTest, BitpackedOutput) { runTest<TBitpacked>(TestParam(GetParam())); }
```

Reply:
Good idea :+1: